hr_employee_lastnames/models/hr_employee.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.hr_employee_firstname.models.hr_employee import UPDATE_PARTNER_FIELDS
+
+_logger = logging.getLogger(__name__)
+
+UPDATE_PARTNER_FIELDS += ['lastname2']
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    lastname2 = fields.Char("Second last name")
+
+    @api.model
+    def _get_name_lastnames(self, lastname, firstname, lastname2=None):
+        return self.env['res.partner']._get_computed_name(
+            lastname, firstname, lastname2)
+
+    def _build_name_in_create(self, vals):
+        if any([field in vals for field in 'firstname', 'lastname', 'lastname2']):
```

Comment:
```suggestion
        if any(field in vals for field in 'firstname', 'lastname', 'lastname2'):
```

Reply:
@hugho-ad 

This change generates the following error:

`Generator expression must be parenthesized if not sole argument`